nncf/openvino/graph/node_utils.py

```diff
@@ -79,6 +80,9 @@ def is_node_with_bias(
     if node.metatype not in metatypes_with_bias:
         return False
 
+    if node.layer_attributes is None:
```

Comment:
Since some time ago we removed the shape compliance check for bias in the `Add` operation, this method works with an issue in the case of the `MatMul -> Add` pattern. Even if `MatMul` does not have weights but is followed by an `Add` operation with any shape, the method will return True.
This addition is designed to check for layer attributes, which are added only for operations with weights - https://github.com/openvinotoolkit/nncf/blob/develop/nncf/openvino/graph/nncf_graph_builder.py#L218

Reply:
"followed by an Add operation with any shape", do you mean with any constant?

Reply:
Yes. Any shape of the `Add` constant.

Reply:
Here is an example from the `levit-128` model. Highlighted `Add` node was marked as bias before this change.
![image](https://github.com/openvinotoolkit/nncf/assets/32988081/a802fd42-a75c-4483-9391-9d939b8b0a69)

Reply:
Thanks for the explanation. What do you think about adding a comment to this?

Reply:
No problem. The comment was added.